src/OpenTok/OpenTok.php

```diff
@@ -215,9 +215,29 @@ public function createSession($options=array())
      * set to routed); you cannot archive sessions with the media mode set to relayed.
      *
      * @param String $sessionId The session ID of the OpenTok session to archive.
-     * @param String $name The name of the archive. You can use this name to identify the archive.
-     * It is a property of the Archive object, and it is a property of archive-related events in the
-     * OpenTok JavaScript SDK.
+     *
+     * @param array $options A set of options for the archive. This array includes the
```

Comment:
actually on some research, the `@deprecated` annotation is only used for "structural" elements, so that does not include method parameters. lets just leave the old way undocumented, but i'll make sure the implementation supports it. i'll add a comment in the code that says we can remove that support in 3.0 (the next breaking change release version).